libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/tabs/summaryTab.tsx

```diff
@@ -88,22 +89,22 @@ export const SummaryPanel = ({ workflowId }: { workflowId: string }) => {
               })}
               :
             </Text>
-            <Link className="msla-template-quickview-source-code" href={manifest?.sourceCodeUrl} target="_blank">
-              {manifest?.sourceCodeUrl}
+            <Link className="msla-template-quickview-source-code" href={workflowManifest?.sourceCodeUrl} target="_blank">
+              {workflowManifest?.sourceCodeUrl}
               <Open16Regular className="msla-templates-tab-source-code-icon" />
             </Link>
           </div>
         )}
-        {Object.keys(detailsTags).map((key: string) => {
+        {Object.keys(detailsTags).map((key) => {
           return (
             <div className="msla-template-overview-section-detail" key={key}>
-              <Text className="msla-template-overview-section-detailkey">{detailsTags[key]}:</Text>
-              <Text>{manifest.details[key]}</Text>
+              <Text className="msla-template-overview-section-detailkey">{detailsTags[key as Template.DetailsType]}:</Text>
+              <Text>{templateManifest.details[key as Template.DetailsType]}</Text>
             </div>
           );
         })}
       </div>
-      {manifest.tags?.length ? (
+      {templateManifest.tags?.length ? (
```

Comment:
SO both for accelerators and single workflow templates we will always be showing the template tags here not the workflow specific tags... wont we have tags in both the locations?